apis/rust/node/src/node/mod.rs

```diff
@@ -335,16 +421,18 @@ impl Drop for DoraNode {
     #[tracing::instrument(skip(self), fields(self.id = %self.id), level = "trace")]
     fn drop(&mut self) {
         // close all outputs first to notify subscribers as early as possible
-        if let Err(err) = self
-            .control_channel
-            .report_closed_outputs(
-                std::mem::take(&mut self.node_config.outputs)
-                    .into_iter()
-                    .collect(),
-            )
-            .context("failed to close outputs on drop")
-        {
-            tracing::warn!("{err:?}")
+        if !self.detached {
```

Comment:
This does not prevent that inputs are closed, they are just closed later (through the `report_outputs_done` call at the end of the `drop` impl and through the daemon when the node stops).

Reply:
Indeed